### ISSUE TITLE
Can't connect to SharePoint hosted on Azure

### ISSUE THREAD

**Issue by miek770** [jasonrollins/shareplum#42]
Hi,

I'm trying to connect to a SharePoint site that's hosted on Azure using [this method](https://shareplum.readthedocs.io/en/latest/tutorial.html#office-365-authentication), but it throws the same error (`Exception: Check username/password and rootsite`, see below) even though I'm sure all 3 of them are correct (double checked & tested in a browser).

I'm told Azure uses SharePoint version 2016 for my site. Am I simply missing something, ex.: do I need to register shareplum on Azure and give it access to my site?

Would you need additional information to point me in the right direction?

Thank you,

Michel

```
$ python
Python 3.7.0 (default, Jun 28 2018, 08:04:48) [MSC v.1912 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from shareplum import Site
>>> from shareplum import Office365
>>> authcookie = Office365('https://****.sharepoint.com', username='****@****.com', password='****').GetCookies()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\ProgramData\Anaconda3\lib\site-packages\shareplum\shareplum.py", line 75, in GetCookies
    sectoken = self.GetSecurityToken(self.Username, self.Password)
  File "C:\ProgramData\Anaconda3\lib\site-packages\shareplum\shareplum.py", line 68, in GetSecurityToken
    raise Exception('Check username/password and rootsite')
Exception: Check username/password and rootsite
```

**Comment by jasonrollins** [comment]
I'm not sure what your specific issue could be.  I have not tried to connect to a SharePoint site on Azure before.

**Comment by miek770** [comment]
Are you aware if others successfully connected to Azure using shareplum?

Is this something you're planning on supporting in the future?

Thanks.

**Comment by jasonrollins** [comment]
I believe this is now fixed in v0.4.2.
